### ISSUE TITLE
Some Feedback & Issues.

### ISSUE THREAD

**Issue by DeMoN100** [TryQuiet/quiet#2562]
Hello first time checking this out I tried to join the community from the website tryquiet.org/join when you hit join community it returns sorry no invitation code has been passed with this url. I would like to join a chat so I may keep in touch & provide more.

_Issues_
I was able to create one channel sucessfully then after that the wheel on the purple bar just keeps spinning & it never creates another channel. Second in the first channel I was able to send one message then after that none of my messages are able to send.

Features
Dark Theme - Standard request every app should have a dark theme especially chat apps. The all white is blinding & a eye strainer.

Customizable Colors UI - Almost everybody likes to customize their themes especially me. A UI being fully customizable with rgb color picker is key to keeping people on your app it makes everyone happy. For example with this photo I like to keep my UI & my keyboard matching colors. I prefer to make things black amoled with red & if I can't do that with your app i'll just move on to the next one that will.

![IMG_20240604_202528](https://github.com/TryQuiet/quiet/assets/8564572/b5ac148d-8179-424a-a745-9ce423740a4a)

Forums/Threads - Discord's style of a thread channel is pretty neat I like how tags can filter out post that don't have the tags & you choose the tag icon. I would like that on here it's a great way to keep track & engage in a single topic while the rest of the chat keeps going on.

Thread / Media/ Files Display - If threads however is designed in similarity to the matrix format where you can just right click or hold on any chat to reply in thread then add a thread list tab. Along with a media & file list tab in a button which can be placed in the top right corner next to the 3 dots to access these 3 tabs. You would hit the button next to the three dots it would pop open a new window with 3 tabs first one threads second one media third one files.

Using photos from matrix to show display for example of implementation.

Threads List
![IMG_20240604_204711](https://github.com/TryQuiet/quiet/assets/8564572/4684bbd6-ad99-4328-903c-4600d2f1858c)

Media Viewer
![IMG_20240604_204034](https://github.com/TryQuiet/quiet/assets/8564572/36cf81f0-9134-449c-8fb6-8398c25455e3)

Files List
![IMG_20240604_204025](https://github.com/TryQuiet/quiet/assets/8564572/fe162933-8b1b-48ee-9c23-838181a61a55)

So you understand the idea for quiet you're looking at the channel you hit the button next to the three dots a window would pop up with these 3 tabs at the top switching which is displayed along with a swipe gesture to switch displays so you don't have to always extend your thumb to the top of the screen to switch tabs.

**Comment by holmesworcester** [comment]
Thanks for all this feedback @DeMoN100! 

* Re: dark mode, it is complete and we are about to release it on desktop, but we still have to do it on mobile.
* We've been thinking that for color customization each community would customize the color of the top bar on mobile, similar to Slack, but that it wouldn't apply to the primary text/icon color. If we let users customize that too, would that be okay? https://www.figma.com/proto/8R3jXuoUxhNyczcClFTrDm/Community-settings%2Fswitcher-%2B-dark-mode-%2B-prototypes?node-id=1371-35888&starting-point-node-id=1371%3A35888&show-proto-sidebar=1
*  Re: threads, we are planning to do it the way Matrix and Slack do it, and there would be a "threads" item above the channel list. What do you think of this? https://www.figma.com/design/ri4AfHBYsrVsP4WRz3P6z4/Threads?node-id=887-24995&t=0wxJBu1GceDlbTlZ-4  

> I was able to create one channel sucessfully then after that the wheel on the purple bar just keeps spinning & it never creates another channel. Second in the first channel I was able to send one message then after that none of my messages are able to send.

Can you send a video of the error you're getting when you create a channel?

**Comment by DeMoN100** [comment]
Yes absolutely the color of text & icons is important every aspect must be customizable so a person can truly feel the theme they make is personal so they will be happy to stay on it. It took me a bit to get figma to load properly so I can see the design of threads. I was a little confused by the layout of if so I saw the threads are listed then you tap one & when it opens it shows the thread topic with 3 replies & you have to hit show more replies to see all of them? If that's so I don't think thats a good idea i'm already in the thread to see the replies why do I need another tap press to see all of the replies the extra step should be eliminated all replies should just be shown in the thread & we scroll or if it's like 10+ replies a swipe gesture to jump to bottom & back to top or perhaps a mini sidebar on right side?

As for asking if you should use up the extended left side of the UI underneath DMs I don't think dms or recent threads should be displayed there. Incase anyone wants to screenshot to show a picture of the community or something you wouldn't want your most recent dms displayed it's too much info displayed. I think personally it would be better to have dms in a separate window as you have that at the top & the button I was mentioning next to the top right for media/file list display.

As for the error that happened very weird when I tried to recreate it for a screen record the problem didn't happen & it worked I don't know why. Is there a matrix room for quiet so I can chat there since the join link does not work?

**Comment by holmesworcester** [comment]
> I was a little confused by the layout of if so I saw the threads are listed then you tap one & when it opens it shows the thread topic with 3 replies & you have to hit show more replies to see all of them? If that's so I don't think thats a good idea i'm already in the thread to see the replies why do I need another tap press to see all of the replies the extra step should be eliminated all replies should just be shown in the thread

I think we're doing what you want: if you tap a thread you will see the full thread, but there's also a "threads" tab where we list *all* threads but trim them down to the first post and a few replies.

> I think personally it would be better to have dms in a separate window as you have that at the top & the button I was mentioning next to the top right for media/file list display.

Okay, understood! I'm not sure if we'll go this route because we are trying to keep things familiar to Slack users, but is there an app that you use that does this well on desktop and mobile? 

> Is there a matrix room for quiet so I can chat there since the join link does not work?

We don't have one yet. Is communicating here okay for now? We could just leave this issue open as a space for feedback?

**Comment by DeMoN100** [comment]
> > I was a little confused by the layout of if so I saw the threads are listed then you tap one & when it opens it shows the thread topic with 3 replies & you have to hit show more replies to see all of them? If that's so I don't think thats a good idea i'm already in the thread to see the replies why do I need another tap press to see all of the replies the extra step should be eliminated all replies should just be shown in the thread
> 
> I think we're doing what you want: if you tap a thread you will see the full thread, but there's also a "threads" tab where we list _all_ threads but trim them down to the first post and a few replies.

Could you show a video of that menu interaction? It's kinda hard to follow on the figma but it looks like to me you see the thread list then you hit the thread then you get a preview of the thread with a few replies then you have to hit show more replies to see the whole thing. I understand what your saying if there's a threads tab but what does that do differently from a channel just show the most recent replied to threads from any channel?
> 
> > I think personally it would be better to have dms in a separate window as you have that at the top & the button I was mentioning next to the top right for media/file list display.
> 
> Okay, understood! I'm not sure if we'll go this route because we are trying to keep things familiar to Slack users, but is there an app that you use that does this well on desktop and mobile?

An app with a dm icon in the top right corner to open your dms? Uh well no I don't but it's good to try something different then copy & paste what other chat apps do you don't need to keep the same layout for people to be comfortable. With that being said it's always a good idea to look for improvements to the design then to replicate.

For example I dislike how discord also guilded has a over padded bottom bar & matrix a three tap press to get to the media & files tab, Why not make it convenient when people are in a chat room people don't want to keep swiping up to find pics or other media so why not have a media & file listing in a convenient place for people to easily one tap & get it? As for a thread tab cool but now we would have 4 icons at the top of the chat.

![Discord Bottom Bar](https://github.com/TryQuiet/quiet/assets/8564572/40498e08-ad87-4f8d-8a94-24889803b62c)
I cut off a bit of the over padding but if you've tried discord then you know how big that bottom bar is on the phone it's too big.

A thread button, a dm button, a file/media button, & the three dots for settings & other menus. Well why not condense the dm button, the thread tab, & the media/file all to one button? It would be one button next to the three dots you hit that button it opens a new window which shows 3 tabs in it threads, media, & files. Then you have the 3 dots button next to it for settings. That would be much more convenient because the main things people would want to look for are easily in one place. Looking for a meme top right button looking for a thread top right button need to read a dm the top right button. One button to rule them all would be convenient. Then you could add a search to the top right so the three in top right would be a search eye glass, a square, & a three dots.
 
> 
> > Is there a matrix room for quiet so I can chat there since the join link does not work?
> 
> We don't have one yet. Is communicating here okay for now? We could just leave this issue open as a space for feedback?

Yeah sure I can look here more often. I'm quicker to responding via matrix if you ever need me to immediately answer you can always find me here.
https://matrix.to/#/!swNNeGRoTSBBwpVwIH:matrix.calitabby.com?via=matrix.calitabby.com

I look forward to seeing more & providing feedback as I always want the best for a new chat app & to hope it's done right from the jump. :)
